stack_helpers.sh

```diff
@@ -163,6 +163,7 @@ function set_no_modules_path() {
   local prefix=${PREFIX:-${HPC_OPT:-"/usr/local"}}
   export PATH=$prefix/bin${PATH:+:$PATH}
   export LD_LIBRARY_PATH=$prefix/lib${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}
+  export LD_LIBRARY_PATH=$prefix/lib64${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}
```

Comment:
Brute force ... lib64 may not even exist on the system, and LD_LIBRARY_PATH is definitely set because of the line above. But doesn't matter, it works just fine.